.nvmrc

```diff
@@ -0,0 +1 @@
+11
```

Comment:
does this tell travis what node version to use?

Reply:
It's for netlify ... as far as I can see it's the only way to specify the node version there ... travis uses the `.travis.yml`